### PR TITLE
cart page is added which contains products and price details.

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
cart page is added which contains product details and price amount.

### CODE REVIEW

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  }
+  
+  .product-links a:hover {
+    color: var(--main-color);
+  }
+  .butoon-wishlist{
+    align-self: center;
+    color: white;
+    font-weight: bold;
+    padding:5px 60px;
```
Comment (xsaurabhsuthar) [inline cart/cart.css]: Should use relative unit instead of pixels.

File: cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  
+  }
+  .price-details h4{
+       color: var(--p-details);
+  }
+  .butoon-place{
+    align-self: center;
+    color: white;
+    font-weight: bold;
+    padding:20px 60px;
```
Comment (xsaurabhsuthar) [inline cart/cart.css:157]: Should use relative units

File: cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                <div class="cart-arrival-card">
+                    <div class="cart-product-img">
+                        <img src="https://images-na.ssl-images-amazon.com/images/I/41IMFnHvpkL._SX328_BO1,204,203,200_.jpg"
+                            alt="">
+                    </div>
+                    <div class="cart-product-details">
+                        <span class="product-catagory"> <b>catagory-</b>Self help</span>
+                        <h2><a href="">The Lifestyle Investor</a></h2>
+                        <p>The 10 rules for life</p>
+                        <div class="product-price"><small>₹96.00</small>₹319</div>
```
Comment (xsaurabhsuthar) [inline cart/cart.html:48]: Use font-size class instead of using small or any text formatting HTML tags.

**Review by xsaurabhsuthar — APPROVED** [review]

File: cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  .product-links {
+    text-align: right;
+  }
+  
+  .product-links a {
+    display: inline-block;
+    margin-left: 5px;
+    color: #a8a8a8;
+    transition: 0.3s;
+    font-size: 35px;
```
Comment (Tarandeep-s1ngh) [inline cart/cart.css]: Suggestion: It's better to use relative units like "rem"

**Review by Tarandeep-s1ngh — APPROVED** [review]

**Pull request merged**